slick/src/sphinx/queries.rst

```diff
@@ -47,6 +47,9 @@ the classes ``AnyExtensionMethods``, ``ColumnExtensionMethods``,
    ``==`` for comparing two values for equality and ``=!=`` instead of ``!=`` for inequality.
    This is necessary because these operators are already defined (with unsuitable types and
    semantics) on the base type ``Any``, so they cannot be replaced by extension methods.
+   Similarly, to combine a Scala ``String`` with a ``Rep[String]`` or ``ConstColumn[String]``, 
+   don't use ``str + rep`` syntax, since Scala provides a universal ``+`` operator for strings
+   that works with ``Any``.  Instead use slick's ``++`` operator: ``(str: Rep[String]) ++ rep``
```

Comment:
Well if the left hand side is a scala string, then it //must// be explicitly cast.
`String.++` is defined (via `TraversableLike`) so you'd get a different error.